DSLink.UWP/UWPPlatform.cs

```diff
@@ -0,0 +1,15 @@
+using DSLink.Connection;
+using PCLStorage;
+
+namespace DSLink.UWP
+{
+    public class UWPPlatform
+    {
+        public static void Initialize()
+        {
+            Configuration.StorageBaseFolder = FileSystem.Current.LocalStorage;
+            Websockets.Universal.WebsocketConnection.Link();
```

Comment:
This looks weird. It looks like a method with a side effect, but there's no web in there?